app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from __future__ import annotations
```

Comment:
Is this import necessary? everything should work without it. But if you want to keep it you must add blank line between. Also list imports from app.errors in one row